lettermgmt/models/letter_type.py

```diff
@@ -1,35 +1,23 @@
 # -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Parthiv Pate, Tech Receptives, Open Source For Ideas
-#    Copyright (C) 2009-Today Tech Receptives(http://techreceptives.com).
-#    All Rights Reserved
-#
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU General Public License as published by
-#    the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU General Public License for more details.
-#
-#    You should have received a copy of the GNU General Public License
-#    along with this program.  If not, see http://www.gnu.org/licenses/.
-#
-##############################################################################
-from openerp.osv import fields, orm
+# Copyright (C) 2009-Today Tech Receptives(http://techreceptives.com).
+# Parthiv Pate, Tech Receptives, Open Source For Ideas
+# Hardikgiri Goswami <hardikgiri.goswami@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from openerp import models, fields
 
-class letter_type(orm.Model):
+
+class LetterType(models.Model):
     """Class to define various types for letters like : envelope,parcel,
     etc."""
     _name = 'letter.type'
     _description = "Letter Type"
-    _columns = {
-        'name': fields.char('Type', required=True),
-        'code': fields.char('Code', required=True),
-    }
-    _sql_constraints = [('code_uniq', 'unique(code)', 'Code must be unique !')]
+
+    name = fields.Char('Type', required=True)
+    code = fields.Char('Code', required=True)
```

Comment:
There's no need for the explicit field string `'Code'`.
Maybe also remove field string `'Type'`, since the field is supposed to be the name of the letter type, just `'Name'` makes sense.